docs/source/release_notes.rst

```diff
@@ -6,15 +6,17 @@ Release Notes
 Future Release
 ==============
     * Enhancements
+        * Refactor build_features code path for performance (:pr:`2224`)
     * Fixes
     * Changes
+        * Add tests/profiling/dfs_profile.py (:pr:`2224`)
```

Comment:
This should probably be in the testing changes section.

Reply:
will do